src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -1698,6 +1698,29 @@ export interface Singularity {
    */
   getUpgradeHomeCoresCost(): number;
 
+  /**
+   * Get Requirements for Company Position.
+   * @remarks
+   * RAM cost: 2 GB * 16/4/1
+   *
+   *
+   * This function will return an object that contains the requirements for
+   * a specific position at a specific country.
+   *
+   * @example
+   * ```js
+   * // NS2:
+   * const COMPANY_NAME = "ECorp";
+   * const POSITION = "Chief Executive Officer";
```

Comment:
Extremely minor gripes, but the // NS2 comment will also be removed from the "ns2" (js) examples, since all examples will just be for regular js scripts (the script type formerly known as ns2). Also variable names in examples should use camelCase for consistency (I wouldn't normally kick back a PR for this, just adding this comment because there are already other changes requested)